### ISSUE TITLE
Using --minloglevel argument in v0.1.90+ causes an error

### ISSUE THREAD

**Issue by jasonnance** [google/sentencepiece#492]
I'm calling `spm_train` via the Python wrapper like so:

```python
# Actual parameters differ -- these are just for the example
texts = ["test", "test2"]
model_path = "/tmp/spm"
vocab_size = 30

with tempfile.NamedTemporaryFile(mode="w") as f:
    f.write("\n".join(texts))
    f.flush()

    spm.SentencePieceTrainer.train(
        f"--input={f.name} --model_prefix={model_path} --vocab_size={vocab_size} --minloglevel=2"
    )
```

Since v0.1.90 (still works up to v0.1.86, the last version until then on PyPI), I get the following error back from spm_train:

```
Not found: unknown field name "minloglevel" in TrainerSpec.
```

Is this a regression or do I need to change my invocation to account for changes in v0.1.90?  Thanks!

**Comment by taku910** [comment]
Confirmed. Fixed in the head.

**Comment by taku910** [comment]
Fixed in v0.1.92.  New pip package is also available.

**Comment by zerotosingularity** [comment]
The v0.1.92 package seems to be 'yanked' from pypi.org. Is there a workaround or eta on the new version?

**Comment by jph00** [comment]
@taku910 would you be able to provide a status update on providing a new pypi build?

(If there's a blocker that the community might be able to help with, could you let us know what it is, and hopefully someone might be able to provide a PR to help sort it out.)

**Comment by taku910** [comment]
Fixed it in v0.19.3

**Comment by jph00** [comment]
@taku910 FYI I don't see it on pypi yet.
